js/main.js

```diff
@@ -20,10 +23,13 @@ function handleSubmit(e) {
     notes: notesInput.value,
     nextEntryId: data.nextEntryId
   };
+  var latestEntry = createEntry(newInput);
+  userEntries.appendChild(latestEntry);
   data.entries.push(newInput);
```

Comment:
you want to use `unshift` here instead because new entries should be added to the front of the array so that when they are loaded into the DOM they show as newest entries first